packages/ketcher-core/src/domain/helpers/attachmentPointCalculations.ts

```diff
@@ -130,16 +130,27 @@ export function getSearchFunction(
       x: Math.round(zoomedCoordinateOfSecondPoint.x) + canvasOffset.x,
       y: Math.round(zoomedCoordinateOfSecondPoint.y) + canvasOffset.y,
     };
-    const newPoint = document.elementFromPoint(
-      newPointCoord.x,
-      newPointCoord.y,
-    ) as Element;
-
-    let newAngle;
-    if (newPoint === monomer.renderer?.bodyElement?.node()) {
-      newAngle = initialAngle;
-    } else {
-      newAngle = initialAngle - 180;
+    let newAngle: number = initialAngle;
+    let newPoint: Element | null = null;
+
+    if (monomer.renderer?.bodyElement) {
+      const elementsAtPoint = document.elementsFromPoint(
+        newPointCoord.x,
+        newPointCoord.y,
+      );
+
+      for (const element of elementsAtPoint) {
+        if (element === monomer.renderer.bodyElement.node()) {
+          newPoint = element;
+          break;
+        }
+      }
```

Comment:
I would recomend using Array.some here instead, which should be more concise.

eg: `const isMonomerAtPoint = elementsAtPoint.some(element =>element  === newPoint ); 
      if(isMonomerAtPoint ){
      newAngle = initialAngle;
      } else {
        newAngle = initialAngle - 180;
      } `